front/app/containers/IdeasNewPage/index.tsx

```diff
@@ -417,7 +418,12 @@ export default withRouter((inputProps: InputProps & WithRouterProps) => {
     name: 'dynamic_idea_form',
   });
 
-  if (isDynamicIdeaFormEnabled) {
+  const project = useProject({ projectSlug: inputProps.params.slug });
+
+  if (
+    isDynamicIdeaFormEnabled ||
+    project?.attributes.participation_method === 'native_survey'
+  ) {
```

Comment:
Same here - perhaps if we have a new property on the project attributes that is more general, we won't have to use native_survey here directly.

Reply:
Without `isDynamicIdeaFormEnabled` native surveys cannot work currently. We need a discussion about this, as native surveys is core functionality, while dynamic idea forms are currently not part of the core.